library/src/c_api.rs

```diff
@@ -96,9 +89,8 @@ pub extern "C" fn shorebird_active_patch_number() -> *mut c_char {
 /// Return the path to the active patch for the app, or NULL if there is no
 /// active patch.
 #[no_mangle]
-// rename to shorebird_patch_path
-pub extern "C" fn shorebird_active_path() -> *mut c_char {
-    let version = updater::active_patch();
+pub extern "C" fn shorebird_next_boot_patch_path() -> *mut c_char {
+    let version = updater::next_boot_patch();
```

Comment:
```suggestion
    let patch_number = updater::next_boot_patch();
```

nit but version might not be the most descriptive name for this 🤷